src/loader/LoadDataAction.cpp

```diff
@@ -259,12 +259,12 @@ Volume* LoadDataAction::run(bool convert)
 
     //needs adjustement of min and max value
     //for now disabled
```

Comment:
Add a TODO comment here. Are you adding all of these as issues in GitHub? I would reference the issue number in the comment